content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -81,9 +81,9 @@ throughout the lifecycle of the pod and can be started and stopped independently
 main container. Unlike [init containers](/docs/concepts/workloads/pods/init-containers/),
 sidecar containers support [probes](/docs/concepts/workloads/pods/pod-lifecycle/#types-of-probe) to control their lifecycle.
 
-These containers can interact directly with the main application containers, sharing
-the same network namespace, filesystem, and environment variables. They work closely
-together to provide additional functionality.
+Sidecar containers can interact directly with the main application containers, sharing
+the same network namespace, and filesystem. They work closely together to
+provide additional functionality.
```

Comment:
Try this:
```suggestion
Sidecar containers can interact directly with the main application containers, because
like init containers they always share the same network, and can optionally also share
volumes (filesystems).
```

Reply:
This is much better, as it also clarifies that the filesystem is not shared by default, but can be shared by configuring volumes. This is a solution to the issue I opened :)

Reply:
thanks @sftim , it actually hits the nail on the head by getting into the nuances of how init containers and volumes work, something I kind of glossed over in my original edit. It's a sharper picture that'll definitely help clear up any confusion for folks diving into Kubernetes.

Reply:
@tnyeanderson thanks for the insight !
I'm going to commit it right away!